### ISSUE TITLE
Compatibility with fish 2.0.0

### ISSUE THREAD

**Issue by enricobacis** [danhper/fundle#25]
Ubuntu 14.04 has fish 2.0.0, and fundle is not working with that version.
We should try to diagnose why it is not working, since ubuntu-14.04-based distros are still widely used.

**Comment by danhper** [comment]
Hi, sorry for the late reply and thanks for reporting.

I know fundle is not working with fish 2.0 and 2.1.
I am not perfectly sure what we would need to make it compatible,
but I think I am not using any "new" features, so it may be worth giving it a shot.

I will try to see how hard it might be to make it compatible.

**Comment by alexkreidler** [comment]
+1 to this issue. I have fish version 2.0 on Ubuntu 14.04. I would really appreciate support for this version. I can post error logs if it would help. Thanks!

**Comment by dagoss** [comment]
I'm not having this issue with Fish 2.2 and Linux Mint 18.1. Fundle is working fine for me!

**Comment by enricobacis** [comment]
@dagoss Fish 2.2 is working for everyone, but version available in the 14.04 repositories is Fish 2.0, which for some reason is not compatible with fish. One has to add the ppa to update fish to version 2.2 to get fundle working, but it would be nice to know why it is not working with 2.0 and, possibly, to fix it.

**Comment by simonszu** [comment]
Found this issue because i tried to install fish on Raspbian for Raspberry Pis. 
Precompiled packages provide only fish 2.1.0, and compiling fish on my own on a small ARM box is a PITA. 
So this is my error when trying to load fundle on fish 2.1.0:
```
fish: Could not expand string “$name_path[2]”
/home/simonszu/.config/fish/functions/fundle.fish (line 253):           __fundle_profile_or_run $profile __fundle_load_plugin $name_path[1] $name_path[2] $fundle_dir $profile
                                                                                                                                                  ^
in function “__fundle_init”,
        called on line 370 of file “/home/simonszu/.config/fish/functions/fundle.fish”,
        with parameter list “”

in function “fundle”,
        called on line 2 of file “/home/simonszu/.config/fish/config.fish”,
        with parameter list “init”

in . (source) call of file “/home/simonszu/.config/fish/config.fish”,
        called on standard input,
```
Maybe it helps to fix this issue.
